.github/ISSUE_TEMPLATE/01-bug.yml

```diff
@@ -108,6 +108,7 @@ body:
         - resource:database_role
         - resource:dynamic_table
         - resource:email_notification_integration
+        - resource:execute
```

Comment:
I totally forgot about adding new resources here. Can I just add them or do I need to add the matching labels first? (e.g. for new functions and procedures)

Reply:
Hmm, that is a good question, and I couldn't find an answer for it. Maybe we can see after merging this one because I haven't added labels yet in the repo.

Reply:
Note: we have SNOW-1831333 for this.